cypress/components/button-toggle/button-toggle.cy.tsx

```diff
@@ -94,40 +97,6 @@ context("Testing Button-Toggle component", () => {
       });
     });
 
-    describe("rounded corners", () => {
```

Comment:
Rounded corners will conditionally change if the prop will be passed and css variable for design token will be different.

Reply:
Okay that's fine, but we should be creating tests for either scenario instead of just deleting the rounded corners tests completely, imo

Reply:
After doing some digging it looks like this was just failing because the variants were getting messed up. The `ButtonToggleComponent` we use on the failing test returns multiple `ButtonToggle`'s which are wrapped by a `<div />` and not a `ButtonToggleGroup`. If we ensure the `variant` prop in `button-toggle.component.tsx` is default like so `variant="default"` the tests pass. Alternatively, if you would like you can wrap the `ButtonToggle`s rendered in `ButtonToggleComponent` with a `ButtonToggleGroup`, instead of a `<div />` to achieve the same result :)